sdk/servicebus/service-bus/package.json

```diff
@@ -94,6 +96,7 @@
     "assert": "^1.4.1",
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
+    "chai-exclude": "2.0.2",
```

Comment:
I dont see this added dependency being used in the PR

Reply:
It is used now in the tests